src/MatBlazor/Components/MatSlider/BaseMatSlider.cs

```diff
@@ -10,6 +11,10 @@ namespace MatBlazor
     /// <typeparam name="TValue">sbyte, byte, short, ushort, int, uint, long, ulong, char, float, double, decimal, decimal?</typeparam>
     public class BaseMatSlider<TValue> : BaseMatInputComponent<TValue>
     {
+        private TValue valueMin;
+        private TValue valueMax;
+        private TValue step;
```

Comment:
private TValue step is ambiguous.

Reply:
i can agree it could be more specific. do you think that stepValue or valueStep would fit better ?

Reply:
I would say Refer to how the other components do it but MatSlider is ValueMax / ValueMin / Step  & MatNumeric is  Maximum, Minimum and Step.  

I would like to see it consistent across the components or since we only have 2 instances of "Max/Min/Step" that I found we can refer to material.io for the tie breaker?  

**Material-UI uses step, min, max for their Slider - Which I think is best and suggest refactoring the Slider and MatNumeric for this. Not in this PR.**

@SamProf  or @lindespang  have an opinion?

Reply:
@enkodellc I agree!